godot/godot-cpp/godot_headers/gdnative/rid.h

```diff
@@ -37,7 +37,7 @@ extern "C" {
 
 #include <stdint.h>
 
-#define GODOT_RID_SIZE sizeof(void *)
+#define GODOT_RID_SIZE sizeof(int64_t)
```

Comment:
This change is most likely wrong, RID is `int64_t` only in master. It's pointer in Godot 3.2 https://github.com/godotengine/godot/blob/803f6d2388f64d069810554ab1b0a57d6a5a8087/core/rid.h#L60

Reply:
Thanks, I've re-check `gdnative/rid.h` from Godot Engine 3.2.3 and it definitely have `void *`. Wonder why Godot Engine doesn't use `godot_headers` itself.